person.rb

```diff
@@ -1,3 +1,5 @@
+require
+
 class Person < Nameable
```

Comment:
- [ ] This import returns errors when trying to test the functionality. Kindly complete the import here. And also use `require_relative`, instead of `require`.

  ```
  require_relative './decorator.rb'
  ```